tests/artifacts/openebs-ci.go

```diff
@@ -58,12 +58,13 @@ const (
 type LabelSelector string
 
 const (
-	MayaAPIServerLabelSelector           LabelSelector = "name=maya-apiserver"
-	OpenEBSProvisionerLabelSelector      LabelSelector = "name=openebs-provisioner"
-	OpenEBSSnapshotOperatorLabelSelector LabelSelector = "name=openebs-snapshot-operator"
-	OpenEBSAdmissionServerLabelSelector  LabelSelector = "app=admission-webhook"
-	OpenEBSNDMLabelSelector              LabelSelector = "name=openebs-ndm"
-	OpenEBSCStorPoolLabelSelector        LabelSelector = "app=cstor-pool"
+	MayaAPIServerLabelSelector             LabelSelector = "name=maya-apiserver"
+	OpenEBSProvisionerLabelSelector        LabelSelector = "name=openebs-provisioner"
+	OpenEBSLocalPVProvisionerLabelSelector LabelSelector = "name=openebs-localpv-provisioner"
+	OpenEBSSnapshotOperatorLabelSelector   LabelSelector = "name=openebs-snapshot-operator"
+	OpenEBSAdmissionServerLabelSelector    LabelSelector = "app=admission-webhook"
+	OpenEBSNDMLabelSelector                LabelSelector = "name=openebs-ndm"
+	OpenEBSCStorPoolLabelSelector          LabelSelector = "app=cstor-pool"
```

Comment:
cc: @prateekpandey14 - sounds like another place where name=cstor-pool needs to be set. Need to check all the pods generated by CAST are also having same labels.

Reply:
This label will be present, set by runtasks.